indico/modules/events/registration/clone.py

```diff
@@ -120,7 +122,7 @@ def _has_content(self, event):
                 .filter(RegistrationForm.registrations.any(Registration.is_active))
                 .has_rows())
 
-    def _clone_registrations(self, old_form, new_form, field_data_map):
+    def _clone_registrations(self, old_form, new_form, field_data_map, session_blocks_map=None):
```

Comment:
No strong opinion but you're always passing a value for it anyway (even if that value is `None`) so it's not necessary to set a default value

```suggestion
    def _clone_registrations(self, old_form, new_form, field_data_map, session_blocks_map):
```

Reply:
I would keep it just in case in the future somebody needs to call this function and doesn't needs the session_blocks_map, it's not something will be always needed and I had to inject it in the function